pkgs/ffigen/lib/src/code_generator/objc_protocol.dart

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) 2024, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:ffigen/src/code_generator.dart';
+
+import 'binding_string.dart';
+import 'utils.dart';
+import 'writer.dart';
+
+class ObjCProtocol extends NoLookUpBinding with ObjCMethods {
+  final superProtos = <ObjCProtocol>[];
```

Comment:
Nit here and in other placces: Maybe don't abbreviate protocol to proto, Protocol is only 3 characters longer than proto, and proto can mean protocol buffer.

Reply:
Done